### PR TITLE
Correcting storage duration information for build exports

### PR DESCRIPTION
**Pull request by 123sarahj123**
See this comment and comments that follow it:
https://3.basecamp.com/3453178/buckets/27608512/messages/5965574604#__recording_6170797891

If a customer does not configure their own bucket for build exports, we store it in our own S3 bucket.

Although the current implementation stores it for 18months, 18 months is just a safe-guard for this feature _which will be reduced to 12 months._

We are only advertising that we store it for an additional 12 months (not 18 months) to customers.

- [x] Have updated the build retention page flow chart to show we only store for an additional 12 months (not 18 months)
- [x] Have updated the build exports page to say 12 months (not 18 months) 

image

image

### CODE REVIEW

File: images/docs/pipelines/build_retention/PB_WO6179144.pdf
Comment (mbelton-buildkite) [inline images/docs/pipelines/build_retention/PB_WO6179144.pdf:1]: This has been pulled in accidentally 😆

**Review by mbelton-buildkite — COMMENTED** [review]
Content is all good, but there's an extra file that shouldn't be added

**Comment by 123sarahj123** [comment]
@mbelton-buildkite all cleaned up :)

File: images/docs/pipelines/build_retention/build-exports-flow-chart-f4f65e956aafcde1b58292a57c3ae3fe92a63a5c02e45ab01469a917ddf22692.png
Comment (mbelton-buildkite) [inline images/docs/pipelines/build_retention/build-exports-flow-chart-f4f65e956aafcde1b58292a57c3ae3fe92a63a5c02e45ab01469a917ddf22692.png:1]: Sorry, if I missed this the first time, but what's going on with the file name here? It looks like this doubles up `build-exports-flow-chart.png` but isn't used? Also, should the 18 months be changed to 12 months?

Comment (123sarahj123) [reply images/docs/pipelines/build_retention/build-exports-flow-chart-f4f65e956aafcde1b58292a57c3ae3fe92a63a5c02e45ab01469a917ddf22692.png:1]: removed

**Comment by 123sarahj123** [comment]
updated build exports graph

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me :)

**Pull request merged**